mods/ra/rules/vehicles.yaml

```diff
@@ -68,7 +68,7 @@ V2RL:
 	Armor:
 		Type: Heavy
 	Mobile:
-		Speed: 128
+		Speed: 118
```

Comment:
I thought the reasoning behind light tanks being a bit faster than flak trucks is that they can chase down / counter flak trucks (which are much more versatile) in the early game. Is there a consensus between the competitive players that this is needed/wanted (and not just you changing balance to counter a build you don't like to play against)?

Reply:
The original purpose of the speed reduction was so that flaks couldn't chase hinds well. The indirect "buff" to the light tank didn't seem like it would hurt. However, with the range increase as well 1 light tank will chase down and obliterate a whole flak squad. It's a bit too swingy.

There is some contention on this in the community but overall I'd personally rather try it first. The alternative is reverting the speed reduction on the flak truck. I think leaving things as they are would be bad.

Reply:
> The original purpose of the speed reduction was so that flaks couldn't chase hinds well. The indirect "buff" to the light tank didn't seem like it would hurt. However, with the range increase as well 1 light tank will chase down and obliterate a whole flak squad.

This shows how fragile such a balance system can be, how one small change effects many other things.
Another bad side effect with this change is that Lighttanks  are loosing some important speedpoints if you use them for infantry crunshing.
Thats an important ability, i would say most important in the early game.
The soviet units for that is the **apc** with **speed: 142**

Reply:
How about increasing the vision of the flak (only the vision, not fire range), so that they can stay away from threats?

Benefits:
- still nerfed against hinds/BH
- makes the LT less OP against both sov vehicls
- balance the huge map awareness advantage allies can have

Reply:
It also makes flak much better at killing infantry early game and being a spotter for them. It also makes it very significantly more affective versus aircraft. I am not strongly against this change but it is not necessary now and some of these consequences may be undesired. If is going to happen it will not be this release cycle